src/transformers/models/t5/modeling_t5.py

```diff
@@ -1988,6 +2002,8 @@ def forward(
         return_dict: Optional[bool] = None,
         encoder_position_ids_dict: Optional[Dict[str, Tuple[torch.LongTensor,str]]] = None,  # shape of each LongTensor (num_batches, n_input_tokens)
         decoder_position_ids_dict: Optional[Dict[str, Tuple[torch.LongTensor,str]]] = None, # shape of each LongTensor (num_batches, n_input_tokens)
+        encoder_input_scalars_indices: Optional[torch.LongTensor] = None,
```

Comment:
is it a list of tensors in different sizes or a single padded tensor?

Reply:
fixed the typing hint